releasenotes/upgrade.rst

```diff
@@ -142,7 +142,15 @@ The migration scripts usually perform two operations in this order:
    #. Migrate the existing data using the ``updateMany()`` operator.
    #. Update the system information document to the next version.
 
-While migrating large databases, it is possible that the migration script times out, is interrupted or encounters other unforeseen circumstances. In this case, migrations initiated by the ``updateMany()`` operator will continue running in the background. The update of the schema version, however, will not succeed and needs to be performed manually. Follow these steps to do so: 
+While migrating large databases, it is possible that the migration script times out, is interrupted or encounters other unforeseen circumstances. We therefore recommend using the ``screen`` tool to perform the migration:
+
+#. Install the ``screen`` tool or check whether it is installed with ``screen --version``
+#. Start a screen session: ``screen``
+#. Execute the migration script in this screen session
+#. In case the connection breaks, you can reconnect to the session via ``screen -r``
```

Comment:
```suggestion
#. In case the SSH connection breaks, you can connect via SSH again and get access to the running migration process using the command ``screen -r``
```